jobs/config.json

```diff
@@ -358,6 +359,7 @@
       "--env-file=platforms/gce.env", 
       "--env-file=jobs/ci-kubernetes-e2e-gce.env", 
       "--env-file=jobs/ci-kubernetes-e2e-gce-canary.env", 
+      "--publish=gs://kubernetes-release-dev/ci/latest-canarygreen.txt", 
```

Comment:
I'm keeping it to help test regressions in --publish before we release the new image.